# Generated by Django 2.2.9 on 2020-09-17 20:07

import django.contrib.postgres.fields.jsonb
import django.core.validators
from django.db import migrations, models
import uuid


class Migration(migrations.Migration):

    dependencies = [
        ('alunos', '0031_auto_20200915_1142'),
    ]

    operations = [
        migrations.CreateModel(
            name='LogErroAtualizacaoEOL',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('criado_em', models.DateTimeField(auto_now_add=True, verbose_name='Criado em')),
                ('alterado_em', models.DateTimeField(auto_now=True, verbose_name='Alterado em')),
                ('uuid', models.UUIDField(default=uuid.uuid4, editable=False, unique=True)),
                ('codigo_eol', models.CharField(max_length=7, validators=[django.core.validators.MinLengthValidator(7)], verbose_name='Código EOL do Aluno')),
                ('cpf', models.CharField(blank=True, max_length=11, null=True, validators=[django.core.validators.RegexValidator(message='Necessário 11 digitos', regex='^\\d{11}$')], verbose_name='CPF')),
                ('nome', models.CharField(blank=True, max_length=255, null=True, verbose_name='Nome do Responsável')),
                ('erro', django.contrib.postgres.fields.jsonb.JSONField(blank=True, default=dict, verbose_name='Erro')),
            ],
            options={
                'verbose_name': 'Log de Erro na atualização no EOL',
                'verbose_name_plural': 'Logs de Erros em Atualizações no EOL',
            },
        ),
    ]
